tests/data/codelist_wildcard/scenario/scenario.yaml

```diff
@@ -0,0 +1 @@
+- scen_*
```

Comment:
Could make sense to start putting codelist-validation test data into an own subfolder

Reply:
I cleaned up the complete test data folder in #399. Once that PR is merged, I'll rebase this one and move this file.